tests/modules/js-and-css/optimize-embeds/optimize-oembeds-test.php

```diff
@@ -0,0 +1,101 @@
+				'<blockquote class="tiktok-embed" cite="https://www.tiktok.com/@mdmotivator/video/7203838800164916486" data-video-id="7203838800164916486" data-embed-from="oembed" style="max-width: 605px;min-width: 325px;" > <section> <a target="_blank" title="@mdmotivator" href="https://www.tiktok.com/@mdmotivator?refer=embed">@mdmotivator</a> <p>I tried giving $2000 iPhone away, BUT then this happened 🥺❤️ <a title="iphone" target="_blank" href="https://www.tiktok.com/tag/iphone?refer=embed">#iphone</a> <a title="mom" target="_blank" href="https://www.tiktok.com/tag/mom?refer=embed">#mom</a> <a title="lent" target="_blank" href="https://www.tiktok.com/tag/lent?refer=embed">#lent</a> </p> <a target="_blank" title="♬ original sound - Zachery Dereniowski" href="https://www.tiktok.com/music/original-sound-7203838814572628741?refer=embed">♬ original sound &#8211; Zachery Dereniowski</a> </section> </blockquote> <script data-lazy-embed-src="https://www.tiktok.com/embed.js" async ></script>',
+			),
+			// WordPress.tv embed.
+			array(
+				"<iframe title=\"VideoPress Video Player\" aria-label='VideoPress Video Player' width='500' height='281' src='https://video.wordpress.com/embed/Zv05OzFV?hd=1&amp;cover=1' frameborder='0' allowfullscreen allow='clipboard-write'></iframe><script src='https://v0.wordpress.com/js/next/videopress-iframe.js?m=1674852142'></script>",
+				"<iframe loading=\"lazy\" title=\"VideoPress Video Player\" aria-label='VideoPress Video Player' width='500' height='281' src='https://video.wordpress.com/embed/Zv05OzFV?hd=1&amp;cover=1' frameborder='0' allowfullscreen allow='clipboard-write'></iframe><script data-lazy-embed-src=\"https://v0.wordpress.com/js/next/videopress-iframe.js?m=1674852142\" ></script>",
+			),
+
+			// Scribd embed.
+			array(
+				'<iframe loading="lazy" title="ACLS Practice Test2" class="scribd_iframe_embed" src="https://www.scribd.com/embeds/135593338/content" data-aspect-ratio="0.7729220222793488" scrolling="no" id="135593338" width="500" height="750" frameborder="0"></iframe><script type="text/javascript">          (function() { var scribd = document.createElement("script"); scribd.type = "text/javascript"; scribd.async = true; scribd.src = "https://www.scribd.com/javascripts/embed_code/inject.js"; var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(scribd, s); })()        </script>',
+				'<iframe loading="lazy" title="ACLS Practice Test2" class="scribd_iframe_embed" src="https://www.scribd.com/embeds/135593338/content" data-aspect-ratio="0.7729220222793488" scrolling="no" id="135593338" width="500" height="750" frameborder="0"></iframe><script type="text/javascript">          (function() { var scribd = document.createElement("script"); scribd.type = "text/javascript"; scribd.async = true; scribd.src = "https://www.scribd.com/javascripts/embed_code/inject.js"; var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(scribd, s); })()        </script>',
```

Comment:
```suggestion
				null, // No change.
```

Reply:
interesting, this one immediately inserts it's inject script, I wonder if we can handle this case.